src/main.py

```diff
@@ -342,4 +343,14 @@ def handle_bad_gateway(e):
 if __name__ == '__main__':
     # This is used when running locally. Gunicorn is used to run the
     # application on Google App Engine. See entrypoint in app.yaml.
-    app.run(host='127.0.0.1', port=8080, debug=True)
+
+    # If the 'background' command line argument is given:
+    #    python main.py background &
+    # then run in non-debug mode, as debug mode can't be backgrounded
+    # but debug mode is useful in general (as auto reloads on change)
+    if (len(sys.argv) > 1 and sys.argv[1] == 'background'):
+        # Turn off HTTPS redirects (automatically turned off for debug)
+        talisman.force_https=False
+        app.run(host='127.0.0.1', port=8080)
```

Comment:
Pinning `host` to `127.0.0.1` or `localhost` will make it difficult to run in a Docker container or VM when the intent is to access from the host machine. Should we use `0.0.0.0` instead?

Reply:
If we want to run docker, then perhaps we should look at this - as we're discussing in #933 

For now, I'm going to resolve this, as `127.0.0.1` is not something introduced in this PR so think changing that is separate issue to what we're trying to address here.